### PR TITLE
add the patchArticleVotes function. do two tests for the function and…

### PR DESCRIPTION
**Pull request by aloush-dev**
Ticket Number 5
add the patchArticleVotes function. 
do two tests for the function and one error handling test

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -70,3 +70,55 @@ describe("get api articles", () => {
       });
   });
 });
+
+describe("patch api articles", () => {
+  test("201 : should accept an object with a POSITIVE number value. Update the article given in the endpoint with the amount of votes given in the article. ", () => {
```
Comment (daviidbartlett) [inline __tests__/app.test.js]: not a 201, you've not created anything

File: controllers/article-controller.js [hunk 2]
```diff
@@ -1,16 +1,35 @@
...
+      res.status(200).send({ article: data[0] });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.patchArticleVotes = (req, res, next) => {
+  if(!req.body.hasOwnProperty("inc_votes")){
+    res.status(400).send("Wrong Object Type")
```
Comment (daviidbartlett) [inline controllers/article-controller.js]: can we send everything back in an object for consistency.

File: controllers/article-controller.js [hunk 3]
```diff
@@ -1,16 +1,35 @@
-const { fetchArticlesByID } = require("../models/article-model");
-
+const res = require("express/lib/response");
```
Comment (daviidbartlett) [inline controllers/article-controller.js]: what is this? why have you got it?

File: controllers/article-controller.js [hunk 4]
```diff
@@ -1,16 +1,35 @@
...
-      .catch((err) => {
-        next(err);
-      });
-  };
-  
\ No newline at end of file
+  fetchArticlesByID(req.params)
+    .then((data) => {
+      if (data.length === 0) {
+        res.status(404).send("Not Found");
```
Comment (daviidbartlett) [inline controllers/article-controller.js]: your code is going to get very repetitive if you keep up this pattern. Take a look at the error handling notes to make use of next

**Review by dappernerddesigns — APPROVED** [review]
Nicely done, merge when ready

**Pull request merged**